jarviscli/plugins/flights.py

```diff
@@ -0,0 +1,201 @@
+from xmlrpc.client import getparser
+from plugin import plugin, require
+
+from colorama import Fore
+import requests
+#from ryanair import Ryanair
+from FlightRadar24.api import FlightRadar24API
+
+@require(network=True)
+@plugin('flights')
+class Flights():
+    def __call__ (self,jarvis,s):
+        self.flights(jarvis,s)
+    def flights(self,jarvis, s):
+        fr_api = FlightRadar24API()
+        jarvis.say("Welcome to Flight module."+ "\n")
+        self.newRadarApp(jarvis,s)
+
+
+    def newRadarApp(self,jarvis, s):
+        url='https://app.goflightlabs.com/advanced-real-time-flights?access_key='
```

Comment:
Yeah, API keys in code is a bit of a problem. Apart from that looks good so far, I'll come back to this later.
I really should finish + document the central API management functionality.